src/robotide/__init__.py

```diff
@@ -109,15 +120,17 @@ def _show_old_wxpython_warning_if_needed(parent=None):
     title = 'Please upgrade your wxPython installation'
     message = ('RIDE officially supports wxPython 2.8.12.1. '
                'Your current version is %s.\n\n'
-               'Older wxPython versions are known to miss some features used by RIDE. '
+               'Older wxPython versions are known to miss some features used b\
+y RIDE. '
                'Notice also that wxPython 3.0 is not yet supported.\n\n'
                'wxPython 2.8.12.1 packages can be found from\n'
-               'http://sourceforge.net/projects/wxpython/files/wxPython/2.8.12.1/.'
+               'http://sourceforge.net/projects/wxpython/files/wxPython/2.8.12\
+.1/.'
```

Comment:
Cutting lines like this makes absolute no sense. The source code gets a lot harder to read. Using a multiline string would be a better solution here.